cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectBuild.java

```diff
@@ -235,13 +265,14 @@ public KafkaConnectDockerfile generateDockerfile()  {
      * Generates builder Pod for building a new KafkaConnect container image with additional connector plugins
      *
      * @param isOpenShift       Flag defining whether we are running on OpenShift
+     * @param isBuildahBuild    Flag defining whether we should use Buildah or not
```

Comment:
this should be clear, "or not" actually means Kaniko, because if it's "false" it can't be nothing.